core/backend/src/test/standalone/GeometryStream.test.ts

```diff
@@ -2270,6 +2270,28 @@ describe("BRepGeometry", () => {
     }
   });
 
+  it("offset surfaces test", async () => {
+    const builder = new ElementGeometry.Builder();
+    builder.appendGeometryQuery(Loop.createPolygon([Point3d.create(0, 0, 0), Point3d.create(0, 5, 0), Point3d.create(5, 5, 0), Point3d.create(5, 0, 0), Point3d.create(0, 0, 0)]));
+
+    const onResult: BRepGeometryFunction = (info: BRepGeometryInfo): void => {
+      assert.isTrue(undefined !== info.entryArray && 1 === info.entryArray.length && ElementGeometryOpcode.BRep === info.entryArray[0].opcode);
+      // assert.isTrue(DbResult.BE_SQLITE_OK === createGeometricElemFromSeed(imodel, "0x1d", info.entryArray));

```

Comment:
Do you have plans to uncomment this line and line 2323 in the future, or can delete?

Reply:
I just uncomment as needed when I want to create the element to have an imodel to look at should there be any problem. I can remove if it really offends. :)

Reply:
Nope.